docs/template-blocks.md

```diff
@@ -2,24 +2,24 @@
 
 | Template block name       |  Location / Information                         |   Default template values
 |---                        |---                                              |---
-| top_of page               | Before doctype                                  | Insertion point
+| top_of_page               | Before doctype                                  | Insertion point
 | html_lang                 | value of the HTML lang attribute                | en
 | page_title                | Text inside the `<title>` element               | GOV.UK - The best place to find government services and information
 | head                      | Before closing `</head>` element                | Insertion point
 | body_classes              | Classes to be added to the `<body>` element     | Insertion point
 | body_start                | After opening `<body>` element                  | Insertion point
-| skip_link_message         | After `body_start` block                        | Skip to main content
-| cookie_message            | After `#skiplink-container` div                 | `<p>GOV.UK uses cookies to make the site simpler. <a href="https://www.gov.uk/help/cookies">Find out more about cookies</a></p>`
+| skip_link_message         | Text inside the skip to main content link       | Skip to main content
+| cookie_message            | Text inside the cookie message banner           | `<p>GOV.UK uses cookies to make the site simpler. <a href="https://www.gov.uk/help/cookies">Find out more about cookies</a></p>`
 | header_class              | `<header>` element                              | Set the value of header_class to [with-proposition](docs/usage.md#propositional-title-and-navigation) to show the propositional navigation
 | homepage_url              | URL of anchor element wrapping logo             | https://www.gov.uk/
 | logo_link_title           | Title of anchor element wrapping logo           | Go to the GOV.UK homepage
 | global_header_text        | Text next to the crown image                    | GOV.UK
 | inside_header             | Inside parent `.header-global`                  | Insertion point
-| proposition_header        | Inside parent `.header-wrapper`                 | Insertion point
+| proposition_header        | Inside parent `.header-wrapper`                 | Add a [propositional title and navigation links](docs/usage.md#propositional-title-and-navigation)
 | after_header              | After closing `</header>` element               | Insertion point
-| content                   | After `#global-header-bar`                      | Insertion point
+| content                   | Main content goes in here                       | Insertion point. Note: A wrapper with an ID of #content is needed here for the skiplink to work correctly
```

Comment:
There's a usage guide on skip links we could cross-link: https://github.com/alphagov/govuk_template/blob/master/docs/usage.md#skip-link